<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmarTSzyme</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>140</y>
      <width>481</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="button_trajectory">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="button_topology">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="edit_topology">
       <property name="placeholderText">
        <string>Select Topology</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="edit_trajectory">
       <property name="placeholderText">
        <string>Select Trajectory</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>SmarTSzyme</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Selection of key residues for enzyme engineering</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>481</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="button_load">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_system">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>508</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMeasures">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionMeasures"/>
    <addaction name="actionSimula"/>
    <addaction name="actionReduce"/>
   </widget>
   <addaction name="menuMeasures"/>
  </widget>
  <action name="actionMeasures">
   <property name="text">
    <string>Measures</string>
   </property>
  </action>
  <action name="actionSimula">
   <property name="text">
    <string>Simula</string>
   </property>
  </action>
  <action name="actionReduce">
   <property name="text">
    <string>Reduce</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
